forge-gui/res/cardsfolder/upcoming/twins_of_discord.txt

```diff
@@ -0,0 +1,14 @@
+Name:Twins of Discord
+ManaCost:7
+Types:Creature Eldrazi
+PT:8/6
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigGenericChoice | TriggerZones$ Battlefield | TriggerDescription$ Whenever you attack, choose odd or even. Creatures with mana value of that quality can't block this turn.
+SVar:TrigGenericChoice:DB$ GenericChoice | Choices$ Odd,Even | Defined$ You 
+SVar:Odd:DB$ Effect | StaticAbilities$ CantBlockOdd | SpellDescription$ Odd
+SVar:CantBlockOdd:Mode$ Continuous | EffectZone$ Command | AffectedZone$ Battlefield | Affected$ Creature.cmcOdd | AddHiddenKeyword$ CARDNAME can't block. | Description$ Creatures with odd mana value can't block this this.
+SVar:Even:DB$ Effect | StaticAbilities$ CantBlockEven | SpellDescription$ Even
+SVar:CantBlockEven:Mode$ Continuous | EffectZone$ Command | AffectedZone$ Battlefield | Affected$ Creature.cmcEven | AddHiddenKeyword$ CARDNAME can't block. | Description$ Creatures with even mana value can't block this this.
```

Comment:
@tool4ever might be simplfied with `cmcChosenEvenOdd`

Also when Static, i don't like that Hidden Keyword :(

Reply:
I've considered it, but it would need extra support in Effect and Cleanup api so that doesn't seem like a good tradeoff either
the hidden KW needs to be fixed in a big AI rework I think